web/package.json

```diff
@@ -1,67 +1,100 @@
 {
-  "name": "sample-metadata",
-  "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-    "@emotion/react": "^11.10.4",
-    "@emotion/styled": "^11.10.4",
-    "@mui/icons-material": "^5.10.9",
-    "@mui/material": "^5.10.9",
-    "@testing-library/jest-dom": "^5.15.0",
-    "@testing-library/react": "^11.2.7",
-    "@testing-library/user-event": "^12.8.3",
-    "@types/jest": "^26.0.24",
-    "@types/node": "^16.11.7",
-    "@types/react": "^17.0.34",
-    "@types/react-dom": "^17.0.11",
-    "axios": "^0.24.0",
-    "lodash": "^4.17.21",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-grid-layout": "^1.3.4",
-    "react-markdown": "^7.1.0",
-    "react-router-dom": "^6.0.1",
-    "react-scripts": "^4.0.3",
-    "react-syntax-highlighter": "^15.4.4",
-    "remark-gfm": "^3.0.1",
-    "remark-toc": "^8.0.1",
-    "semantic-ui-css": "^2.5.0",
-    "semantic-ui-react": "^2.1.3",
-    "swagger-ui-react": "^4.1.2",
-    "typescript": "^3.9.0",
-    "web-vitals": "^1.1.2"
-  },
-  "proxy": "http://localhost:8000",
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "postbuild": "npm run copy:buildartifacts",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject",
-    "copy:buildartifacts": "rm -rf ../api/public/ && mkdir -p ../api/public && cp -r build/* ../api/public/"
-  },
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "react-app/jest"
-    ]
-  },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
+    "name": "sample-metadata",
```

Comment:
Does the linting formatter cover this file, just noticed this frequently gets flicked between 2 and 4 spaces - would be good to try to reduce unrelated changes in the future.

Reply:
Defs should be 4 spaces - haven't noticed the flicking but will keep an eye on it